modules/auxiliary/voip/telisca_ips_lock_abuse.rb

```diff
@@ -0,0 +1,143 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Telisca IPSLock Abuse',
+      'Description'    => %q{This modules will exploit the vulnerabilities of Telisca IPSLock , in order to lock/unlock IP Phones.  you need to be in the voip vlan and you have to  know the phone name example : SEP002497AB1D4B .  Set ACTION to either LOCK or UNLOCK UNLOCK is the default.},
+      'References'     =>
+       [
+       ],
+      'Author'         =>
+       [
+         'Fakhir Karim Reda <karim.fakhir[at]gmail.com>',
+         'zirsalem'
+       ], 'License'        => MSF_LICENSE,
+      'License'        => MSF_LICENSE,
+      'DisclosureDate' => "Dec 17 2015",
```

Comment:
Hi

This is Date of the vulnerabity discovery , i alerted the editor just today
and he confirm the  vulnerability .

Regards.

2016-01-14 17:03 GMT+00:00 Brent Cook <notifications@github.com>:

> In modules/auxiliary/voip/telisca_ips_lock_abuse.rb
> <https://github.com/rapid7/metasploit-framework/pull/6470#discussion_r49752816>
> :
>
> > +  include Msf::Exploit::Remote::HttpClient
> > +
> > +  def initialize(info = {})
> > +    super(update_info(info,
> > +      'Name'           => 'Telisca IPSLock Abuse',
> > +      'Description'    => %q{This modules will exploit the vulnerabilities of Telisca IPSLock , in order to lock/unlock IP Phones.  you need to be in the voip vlan and you have to  know the phone name example : SEP002497AB1D4B .  Set ACTION to either LOCK or UNLOCK UNLOCK is the default.},
> > +      'References'     =>
> > +       [
> > +       ],
> > +      'Author'         =>
> > +       [
> > +         'Fakhir Karim Reda <karim.fakhir[at]gmail.com>',
> > +         'zirsalem'
> > +       ], 'License'        => MSF_LICENSE,
> > +      'License'        => MSF_LICENSE,
> > +      'DisclosureDate' => "Dec 17 2015",
>
> What does this date refer to? Is this disclosed to the vendor?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rapid7/metasploit-framework/pull/6470/files#r49752816>
> .
>